### ISSUE TITLE
plain html version

### ISSUE THREAD

**Issue by jkone27** [rajasegar/todomvc-htmx#1]
interesting maybe to have just plain html files instead of pug files, to remove dependencies? Thanks for sharing this

**Comment by rajasegar** [comment]
@jkone27 Yes, we can have plain html files with a different view engine

**Comment by TheJaredWilcurt** [comment]
Yes, agreed. If this was just a cultural thing with the HTMX community, and all of the examples on the website used Pug, and most HTMX projects were written in Pug, then I think keeping it makes sense. But that doesn't seem to be the case. This is the main "real world" example I would want to look at to demo how HTMX works, and it has an unnecessary barrier.

**Comment by rajasegar** [comment]
@TheJaredWilcurt There is no such cultural thing, it just happened to be a coincidence. Since most of the Node.js and Express examples were written by me, and during that time my personal preference was Pug, so I decided to write them in Pug. I fully understand your concern here and will try to remove the unnecessary barrier in due course of time.

**Comment by nicam** [comment]
@jkone27 in case you're still interested. I forked, updated and reworked this version to use Nunjucks which is a lot closer to html. => https://github.com/nicam/todomvc-htmx

**Comment by kasir-barati** [comment]
@rajasegar I guess now is better than never, since this issue is still open and I am also puzzled by pug. I just wanted to have a bare minimum working backend in ExpressJS and from there to see what it looks like to go back again to things like handlebars or pug.

**Comment by kasir-barati** [comment]
thanks for the work @nicam. Will check it later.

**Comment by kasir-barati** [comment]
I also wanna share a very interesting [YouTube video](https://www.youtube.com/watch?v=te_lYPEDycc) where he is not using a template engine but [native "html" helper function](https://hono.dev/docs/helpers/html#html) provided by the [hono](https://hono.dev/) framework.

*He is using hono instead of ExpressJS. So I am still interested in seeing something purely with ExpressJS and htmx. No additional unnecessary dependency :).

**Comment by kasir-barati** [comment]
Hey folks, I have implemented a very simple app with ExpressJS (no extra dep) + htmx. Check my repo here and give it a star if this was helpful: https://github.com/kasir-barati/htmx
